src/Frame.cpp

```diff
@@ -692,7 +692,7 @@ void Frame::Thumbnail(std::string path, int new_width, int new_height, std::stri
 
 		// Get pixels
 		unsigned char *pixels = (unsigned char *) thumbnail->bits();
-		unsigned char *mask_pixels = (unsigned char *) mask->bits();
+		unsigned char const *mask_pixels = (unsigned char *) mask->constBits();
```

Comment:
**This** change _will_ actually reduce the memory consumption during thumbnailing, since previously the context _wasn't_ `const`. So `QImage` would have been copying the mask bits here.

Minor nit: My preference would be for this more traditional ordering (especially since it makes the `const` far more obvious):
```suggestion
		const unsigned char *mask_pixels = (unsigned char *) mask->constBits();
```